game/src/main/scala/io/aigar/game/Grid.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace._
+import java.util.Random
+
+object Grid {
+  final val WidthPerPlayer = 100
+  final val HeightPerPlayer = 100
+}
+class Grid(val width: Int, val height: Int) {
+  val random = new Random()
+
+  def randomPosition = {
+    val x = random.nextFloat() * width
+    val y = random.nextFloat() * height
+    new Vector2(x, y)
+  }
+
+  def state = {
+    serializable.Dimensions(width, height)
+  }
+}
```

Comment:
@MicLamb it's a [companion-object](http://stackoverflow.com/a/1755942/395386). Allows us to have something similar to static constants. That change was related to my PR by the way!